rskj-core/src/main/java/co/rsk/config/ConfigLoader.java

```diff
@@ -196,6 +196,9 @@ private static boolean isActualObjectValid(@Nonnull String keyPath, @Nonnull Con
             } else {
                 ConfigValue expectedEntryValue = expectedObject.get(actualEntryKey);
                 String entryKeyPath = prefix + actualEntryKey;
+                if (expectedEntryValue == null) {
```

Comment:
makes sense, I'll try that

Reply:
@illuque-iov, I believe the idea here is to make this generic, meaning that it can be used in any other config if needed

Reply:
hmm ok, not very YAGNI (we hadn't needed it until now), but since this is only loaded at startup I don't see any problem with this extra null check, so no big deal, up to you ;)